dev/com.ibm.ws.concurrent.persistent_fat_errorpaths/fat/src/com/ibm/ws/concurrent/persistent/fat/errorpaths/PersistentExecutorErrorPathsTest.java

```diff
@@ -0,0 +1,296 @@
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Tests for error paths in persistent scheduled executor
+ */
+@RunWith(FATRunner.class)
+public class PersistentExecutorErrorPathsTest {
+
+    private static final LibertyServer server = FATSuite.server;
+    
+    private static final String APP_NAME = "persistenterrtest";
+
+    @Rule
+    public TestName testName = new TestName();
+
+    /**
+     * Runs a test in the servlet.
+     *
+     * @param test Test name to supply as an argument to the servlet
+     * @return output of the servlet
+     * @throws IOException if an error occurs
+     */
+    protected StringBuilder runInServlet(String test) throws Exception {
+        URL url = new URL("http://" + server.getHostname() + ":" + server.getHttpDefaultPort() + "/persistenterrtest?test=" + test);
+        Log.info(getClass(), "runInServlet", "URL is " + url);
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+        try {
+            con.setDoInput(true);
+            con.setDoOutput(true);
+            con.setUseCaches(false);
+            con.setRequestMethod("GET");
+            InputStream is = con.getInputStream();
+            InputStreamReader isr = new InputStreamReader(is);
+            BufferedReader br = new BufferedReader(isr);
+
+            String sep = System.getProperty("line.separator");
+            StringBuilder lines = new StringBuilder();
+
+            // Send output from servlet to console output
+            for (String line = br.readLine(); line != null; line = br.readLine()) {
+                lines.append(line).append(sep);
+                Log.info(getClass(), "runInServlet", line);
+            }
+
+            // Look for success message, otherwise fail test
+            if (lines.indexOf("COMPLETED SUCCESSFULLY") < 0) {
+                Log.info(getClass(), "runInServlet", "failed to find completed successfully message");
+                fail("Missing success message in output. " + lines);
+            }
+
+            return lines;
+        } finally {
+            con.disconnect();
+            Log.info(getClass(), "runInServlet", "disconnected from servlet");
+        }
+    }
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+    	ShrinkHelper.defaultDropinApp(server, APP_NAME, "web");
+        server.startServer();
+    }
+
+    /**
+     * After completing all tests, stop the server.
+     *
+     * @throws Exception
+     */
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (server != null && server.isStarted())
+            server.stopServer(LibertyServer.DISABLE_FAILURE_CHECKING);
```

Comment:
I don't think it's necessary to have special code or a special value for disabling error checking. Instead, we could just pass in the regex `.*` which will match everything without needing special handling